tools/code-analyzer/src/commands/analyzer/index.ts

```diff
@@ -222,12 +228,20 @@ export default class Analyzer extends Command {
 				areEqual: boolean;
 			};
 		} | void
-	): void {
+	) {
+		const { flags } = await this.parse( Analyzer );
+
```

Comment:
`output` argument is simply `flags.output` from the main function. I think it would make sense to pass down all the flags instead of re-parsing the Analyzer.